helix-term/src/commands.rs

```diff
@@ -727,6 +728,42 @@ fn goto_file_end(cx: &mut Context) {
     doc.set_selection(view.id, selection);
 }
 
+fn goto_file(cx: &mut Context) {
+    let action = match cx.count() {
+        2 => Action::HorizontalSplit,
+        3 => Action::VerticalSplit,
+        _ => Action::Replace,
+    };
```

Comment:
This doesn't seemed intuitive to me. Does other editors do this as well? It would be hard to remember whether 2 or 3 is horizontal this way.

Reply:
This is emacs style.

Reply:
I think we can just provide a good default for normal use, and add the count style in the doc. Otherwise we have to add and let user remember another two key/command. But if there is any other suggestion, I can change it.

Reply:
I prefer vim style, which is intuitive, at least easier to remember.

Reply:
I agree that it make sense to use `ctrl-w s/v` to split after opening, but we might open many files at once, in that case it will take a lot effort to split each of them.

Reply:
But vim also have `ctrl-w f` which opens it directly in a split.

Reply:
Make sense, and changed based on the comment.